tests/bud.bats

```diff
@@ -117,6 +117,48 @@ symlink(subdir)"
   expect_output --substring $(realpath "${TESTSDIR}/bud/dockerignore3/.dockerignore")
 }
 
+# Following test must fail since we are trying to run linux/arm64 on linux/amd64
+# Issue: https://github.com/containers/buildah/issues/3712
+@test "build-with-inline-platform" {
+  # Host arch
+  run_buildah info --format '{{.host.arch}}'
+  myarch="$output"
+  for try in amd64 arm64 ppc64le s390x; do
+    if [[ "$otherarch" != "$myarch" ]]; then
+      otherarch="$myarch"
```

Comment:
I think otherarch will always be set to either amd64 or arm64 here.  Could we randomize this, or make the loop bigger and test all three of the other archs?